docs/tutorials/01-intro.md

```diff
@@ -0,0 +1,209 @@
+
+    container=$(buildah from fedora)
+
+Of course you can always just run `buildah from fedora` and then list the containers to see the container name. As mentioned previously it will be fedora-working-container in this case.
+
+See the name:  
+
+    echo $container
+
+What can we do with this new container? Let's try running bash:
+
+    buildah run $container bash
+    
+Notice we get a new shell prompt because we are running a bash shell inside of the container. It should be noted that `buildah run` is not intended for running production containers. It is for helping debug during the build process. 
+
+Let's try running something else:
+
+    buildah run $container java
+
+Oops. Java is not installed. A message containing something like the following was returned.
+
+    container_linux.go:274: starting container process caused "exec: \"java\": executable file not found in $PATH"
+    
+Lets try installing it using:
+    
+    buildah run $container -- dnf install java
+
+The `--` syntax basically says: no more `buildah run` command options after this point. It is required if the command we specify includes command line options which are not meant for buildah. 
+
+Now running `buildah run $container java` will show that Java has been installed. It will return the `Usage`:
+
+## Building a container from scratch
+
+One of the advantages of using `buildah` to build OCI compliant container images is that you can easily build a container image from scratch and therefore exclude unnecessary packages from your image. E.g. most final container images for production probably don't need a package manager like `dnf`. 
+
+Let's build a container from scratch. The special "image" name scratch tells buildah to create an empty container:
+
+    newcontainer=$(buildah from scratch)
+  
+You can see this new empty container by running:
+
+    buildah containers
+  
+Its container name is working-container by default. And it's stored in the `$newcontainer` variable. Notice the image name (IMAGE NAME) is "scratch". This just indicates that there is no real image yet. i.e. It is containers/storage but there is no representation in containers/image. So when we run:
+
+    buildah images
+  
+We don't see the image listed. There is no image. It is an empty container.
+
+So does this container actually do anything? Let's see.
+
+    buildah run $newcontainer bash
+    
+Nope. This really is empty. The package installer `dnf` is not even inside this container. It's essentially an empty layer on top of the kernel. So what can be done with that?  Thankfully there is a `buildah mount` command.
+
+    scratchmnt=$(buildah mount $newcontainer)
+    
+By echoing `$scratchmnt` we can see the path for the overlay image. 
```

Comment:
Will the folks you're targeting know what an "overlay image" is?  Should you add something like:  "...overlay image." to
"...overlay image which links you directly to the containers root file system."

Reply:
or "which gives you a link directly"  I think I like that better.

Reply:
Done